my-spec-project/.specify/scripts/powershell/create-new-feature.ps1

```
@@ -0,0 +1,385 @@
+            $cleanNames = $branches | ForEach-Object {
+                $_.Trim() -replace '^\*?\s+', '' -replace '^remotes/[^/]+/', ''
+            }
+            return Get-HighestNumberFromNames -Names $cleanNames
+        }
+    } catch {
+        Write-Verbose "Could not check Git branches: $_"
+    }
+    return 0
+}
+
+function Get-HighestNumberFromRemoteRefs {
```

PSScriptAnalyzer (1.25.0)

[warning] The cmdlet 'Get-HighestNumberFromRemoteRefs' uses a plural noun. A singular noun should be used instead.

Suggested fix: Singularized correction of 'Get-HighestNumberFromRemoteRefs'

(PSUseSingularNouns)